cfme/tests/v2v/test_v2v.py

```diff
@@ -2,19 +2,20 @@
 import pytest
 
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+pytestmark = [pytest.mark.ignore_stream('5.8')]
 
 
-@pytest.mark.tier(0)
-def test_compute_migration_modal(appliance):
-    """Test takes an appliance and tries to navigate to migration page and open mapping wizard.
-
-    This is a dummy test just to check navigation.
-    # TODO : Replace this test with actual test.
-    """
-    infra_mapping_collection = appliance.collections.v2v_mappings
-    view = navigate_to(infra_mapping_collection, 'All')
-    assert view.is_displayed
-    view = navigate_to(infra_mapping_collection, 'Add')
-    assert view.is_displayed
+def test_migration_ui_enable_disable(appliance):
+    appliance.enable_migration_ui()
```

Comment:
So, this test is a little bit ambiguous in that we don't really know the state of the migration feature at the start, and this `enable_migration_ui` call may not actually be doing anything.

Test should either enable..disable..enable, or should have a fixture that disables during setup.

Reply:
@mshriver initially appliance has the state `Disabled`. So enable-disable works.

Reply:
It does now, but will it always? Will all versions keep this configuration?

Reply:
Good question, not sure of this, I will add fixture to disable it before test.